.github/workflows/manifold.yml

```diff
@@ -33,12 +33,15 @@ jobs:
         patch third_party/assimp/code/CMakeLists.txt < assimp.diff
         mkdir build
         cd build
-        cmake -DCMAKE_BUILD_TYPE=Release -DTHRUST_BACKEND=${{matrix.backend}} .. && make
+        echo Python3 Path $(which python3)
+        cmake -DCMAKE_BUILD_TYPE=Release -DTHRUST_BACKEND=${{matrix.backend}} -DPYTHON_EXECUTABLE=$(which python3) .. && make
```

Comment:
Looks like the cmake step is having trouble. Maybe it needs some kind of `FindPython` script in the CMakeLists.txt?

Reply:
I guess CMake is detecting a wrong version of Python here, which is a common problem: https://pybind11.readthedocs.io/en/stable/faq.html#cmake-doesn-t-detect-the-right-python-version

Will need some trial and error to fix this in the CI.